components/infobox/wikis/tetris/infobox_person_player_custom.lua

```diff
@@ -38,10 +41,57 @@ function CustomPlayer:createWidgetInjector()
 	return CustomInjector()
 end
 
+function CustomInjector:parse(id, widgets)
+	if id == 'role' then
+		return {
+			Cell{name = 'Role', content = {
+				CustomPlayer._createRole('role', _role),
+				CustomPlayer._createRole('role2', _role2)
+			}},
+		}
+	end
+	return widgets
+end
+
 function CustomPlayer:adjustLPDB(lpdbData)
-	lpdbData.extradata.role = _role.role
-	lpdbData.extradata.role2 = _role2.role
+
+	local isplayer = 'true'
+	if CustomPlayer._isNotPlayer() then
+		isplayer = 'false'
+	end
+
+	lpdbData.extradata.isplayer = isplayer
```

Comment:
is that really needed?
why not just update the consumers to use type instead
if it is to be kept use the following instead
```suggestion
	lpdbData.extradata.isplayer = tostring(CustomPlayer._isPlayer())
```
and adjust the `CustomPlayer._isNotPlayer()` function accordingly

Reply:
> is that really needed? why not just update the consumers to use type instead if it is to be kept use the following instead
> 
> and adjust the `CustomPlayer._isNotPlayer()` function accordingly

I've tried replaced the line with **lpdbData.type = lpdbData.type** which caused the player to goes back as staff & talent in portal again (with isPlayer showed **Player** instead of true/false) I assume that <https://liquipedia.net/commons/index.php?title=Module:Portal_players&action=edit> requires isPlayer true/false to get player outside of staff section

![image](https://user-images.githubusercontent.com/88981446/224112983-fc50bd3d-4b96-48cc-983a-0b06382dc4ba.png)

Reply:
Yeah, I added it for legacy reasons, and didn't know how much Tetris was relying on modules that need it. I forgot it was a new wiki